setup.py

```diff
@@ -37,6 +47,22 @@
 
 debug_flags = ['DEBUG_COSTLY_THROW']
 
+version_info = sys.version_info
```

Comment:
does this setup.py script run with python 2? Don't we have the previous syntax error?